pkg/trace/config/config.go

```diff
@@ -116,6 +116,18 @@ type AgentConfig struct {
 
 	// Obfuscation holds sensitive data obufscator's configuration.
 	Obfuscation *ObfuscationConfig
+
+	// RequireTags represents a list of tags required for a root span.
```

Comment:
This is not enough.
```suggestion
	// RequireTags specifies a list of tags which must be present on the root span in order
	// for a trace to be accepted.
```
Same idea on the other field. We need to be clear. Always look at it with the eyes of someone who is completely new to the project and trying to understand what this is.